### PR TITLE
feat(dashboard): enhance OpenShift dashboard

### PR DESCRIPTION
**Pull request by jgbernalp**
Fixes: #246

https://issues.redhat.com/browse/OU-250

https://github.com/sustainable-computing-io/kepler-operator/assets/5461414/21c52a33-f013-4b65-be62-95e0d2f6b3a1

### CODE REVIEW

**Comment by vprashar2929** [comment]
Quick question can we add units to the panel value like Watt or Killowatt? For eg. In the case of `Total Energy Consumption` panel  the value we get is in Killowatt-hour

**Comment by jgbernalp** [comment]
@vprashar2929 yes, I added the units in the total energy consumption panel. Unfortunately other panels, like tables and line charts are limited on that

**Comment by jgbernalp** [comment]
Screenshot 2023-09-25 at 16 47 06

**Comment by vimalk78** [comment]
@jgbernalp can we sort the top10 namespace table by values?

**Comment by jgbernalp** [comment]
@vprashar2929 yes makes sense, i've moved them.

**Comment by fkargbo** [comment]
With regards to Phase 1 of this initiative, looks good to me. Thanks!

**Comment by vprashar2929** [comment]
Just one comment about moving `Total Power Consumption (W) in Namespace` and `Total Power Consumption (kWh per day) in Namespace` panels to the top in `kepler-dashboard-namespace.json`

File: pkg/components/exporter/assets/kepler-dashboard-namespace.json
Comment (sthaha) [inline pkg/components/exporter/assets/kepler-dashboard-namespace.json:1]: @jgbernalp could you please rename the file to `powermon-dashboard-namespace.json`

File: pkg/components/exporter/assets/kepler-dashboard-overview.json
Comment (sthaha) [inline pkg/components/exporter/assets/kepler-dashboard-overview.json:1]: @jgbernalp,
For the PR to pass CI,  we need to rename this to `powermon-dashboard-overview.json` and also update the 
https://github.com/sustainable-computing-io/kepler-operator/blob/v1alpha1/pkg/components/exporter/exporter.go#L87 to use this file. 

I will then create a follow-up PR that deploys both dashboards.

**Review by sthaha — CHANGES_REQUESTED** [review]
The https://github.com/sustainable-computing-io/kepler-operator/blob/v1alpha1/pkg/components/exporter/exporter.go#L87 needs correction.

**Comment by jgbernalp** [comment]
Changes were included by #258

**Pull request closed without merging**